enterprise_access/apps/api/tasks.py

```diff
@@ -113,14 +114,25 @@ def decline_enterprise_subsidy_requests_task(subsidy_request_uuids, subsidy_type
         subsidy_request.decline_reason = SUBSIDY_TYPE_CHANGE_DECLINATION
         subsidy_request.save()
 
-@shared_task(base=LoggedTask)
+
+@shared_task(base=LoggedTaskWithRetry)
 def send_notification_emails_for_requests(
         subsidy_request_uuids,
         braze_campaign_id,
         subsidy_type,
+        braze_trigger_properties,
```

Comment:
nit: It seems like we're passing `{}` for this argument in many places. I'm wondering if it might make sense to add `{}` as a default/fallback value when it's not explicitly provided?

Reply:
fun python thing: If you add `{}` as a default, there will only ever be 1 dictionary for that task in the runtime (so each time you call that function, it would refer to the _same_ dictionary)

A common pattern to avoid this is to set it to None and then have an if-statement to make the var a dict if it is None. I can do that